app/models/operatingsystems/debian.rb

```diff
@@ -59,13 +59,31 @@ def shorten_description(description)
     s.presence || description
   end
 
+  def pxe_file_names(medium_provider)
+    if is_subiquity?
+      {
+        kernel: "vmlinuz",
+        initrd: "initrd",
+      }
+    else
+      super
+    end
+  end
+
   private
 
   # tries to guess if this an ubuntu or a debian os
   def guess_os
     (name =~ /ubuntu/i) ? "ubuntu" : "debian"
   end
 
+  def is_subiquity?
+    return false if guess_os != "ubuntu"
+    return false if major.to_i < 20
+    return false if major.to_i == 20 && minor.to_i <= 2
+    true # Ubuntu release 20.04.3 or newer
+  end
+
```

Comment:
Can you remove this method and simply turn this into a single if statement in the `pxe_file_names` method?

Also rebase to resolve the tests.

Reply:
Done.